test/test_torch.py

```diff
@@ -6180,6 +6180,14 @@ def test_comparison_ops(self):
         for idx in iter_indices(x):
             self.assertEqual(x[idx] >= y[idx], ge[idx] == 1)
 
+    def test_comparison_ops_diff_types(self):
+        self.assertRaisesRegex(RuntimeError, "The output tensor of lt must be a bool",
```

Comment:
nit: it's nicer to use `with self.assertRaisesRegex`:

```
with self.assertRaisesRegex(RuntimeError, "...")
    ...
```

Reply:
Changed